docs/bottle.rst

```diff
@@ -6,14 +6,14 @@ Each :ref:`data line <data lines>` in an exchange bottle file represents a singl
 
 When ctd parameters are encountered within exchange bottle files (e.g. :ref:`CTDPRS (DBAR)`) they represent the corrected values being read by the CTD at the time of bottle closure, usually averaged over some interval.
 
-In bottle files, specific parameters are REQUIRED to be present and have non fill values.
+In bottle files, specific parameters are REQUIRED to be present and have non fill values.  **(What does non fill value mean?  Does this mean no empty values?)**
```

Comment:
The fill value is for missing data, in our case `-999` when there is no data. We are trying to say that these parameters must have some meaningful value.